src/Web.cpp

```diff
@@ -1005,7 +1005,7 @@ void Web_SendWebsocketData(uint32_t client, uint8_t code) {
 		JsonObject entry = object.createNestedObject("trackinfo");
 		entry["pausePlay"] = gPlayProperties.pausePlay;
 		entry["currentTrackNumber"] = gPlayProperties.currentTrackNumber + 1;
-		entry["numberOfTracks"] = gPlayProperties.numberOfTracks;
+		entry["numberOfTracks"] = gPlayProperties.playlist->size();
```

Comment:
fixed them and also the other direct access points to gPlayProperties.playlist. I'll clean them up better in the next PR. There I'll encapsulate the playlist and the functions to access it will take care of invalid playlists.